cupyx/scipy/fftpack/fft.py

```diff
@@ -1,6 +1,40 @@
 import cupy
 from cupy.cuda import cufft
-from cupy.fft.fft import _fft, _default_fft_func
+from cupy.fft.fft import (_fft, _default_fft_func, _convert_fft_type,
+    get_cufft_plan_nd)
+
+
+def get_fft_plan(a, value_type='C2C', axes=None):
```

Comment:
+1 on having this utility. It is useful and makes planning easier than using `get_cufft_plan_nd` directly.

Reply:
Glad you like it! I thing logically it's better to expose the plan API in `cupyx.scipy.fftpack` since we allow explicit plans here, and users of `cupy.fft.fft` do not need it. So, I am thinking maybe it's better to rename `get_cufft_plan_nd` to `_ get_cufft_plan_nd` to mark it for internal use? What do you think?